addons/mail/wizard/mail_compose_message.py

```diff
@@ -49,6 +49,11 @@ def default_get(self, fields):
                 - active_ids: record IDs
                 - default_model or active_model
         """
+        # backward compatibility of context before addition of
+        # email_layout_xmlid field: to remove in 15.1+
+        if self._context.get('custom_layout') and not 'default_email_layout_xmlid' in self._context:
```

Comment:
`'default_email_layout_xmlid' not in self._context` instead?
Feels a bit more readable.